Allan690/DevJobs-Slack-bot
# Configure jest for testing

#### Description
- move some modules to devDependencies
- improve on env configuration
- add jest scripts and configs
- add linter and commands
- add morgan for logging requests
- add nodemon for dev purposes
#### Type of change

Please select the relevant option


#### How Has This Been Tested?


#### How to Test
1. checkout to the branch, delete existing `node_modules`, and do a clean install of dependencies `yarn install`
2. run tests using the command `yarn test`
3. run specific test file using the command `yarn test test-file-path`
4. check linting errors `yarn lint`
5. fix linting errors `yarn lint:fix`

#### Any background context you want to add?


 [Resolves Issue #3]
